.github/workflows/backend-prod-server.yml

```
@@ -70,24 +76,66 @@ jobs:
           PID=$(sudo lsof -t -i :8080 || true)
           if [ -n "$PID" ]; then
             echo "Killing existing process: $PID"
             sudo kill -9 $PID
           fi
-    
-      - name: Run App On Port 8080
-        run: |
-          JAR_NAME="${{ steps.get_jar_name.outputs.JAR_NAME }}"
-          DOWNLOADED_JAR="./downloaded-artifact/$JAR_NAME"
-          TARGET_JAR="/home/ubuntu/app/$JAR_NAME"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)